app/templates/admin/createEvent.html

```diff
@@ -152,17 +154,40 @@ <h1 id="title">{{page_title}}</h1>
           </div>
           <div class='form-group'>
             <label class="form-label" for='inputEventFacilitators'><strong>Facilitators</strong></label>
-              <select class="form-select" multiple id="inputEventFacilitators" name="facilitators" >
-              <!-- make sure we show existing facilitators in case they aren't in the facilitators list anymore -->
-              {% for ef in eventData.facilitators %}
-                {% if ef not in allFacilitators %}
-                  <option value="{{ef.username}}" selected>{{ef.firstName}} {{ef.lastName}}</option>
-                {% endif %}
-              {% endfor %}
-              {% for f in allFacilitators %}
-                <option value="{{f.username}}" {{ "selected" if f in eventData.facilitators }}> {{f.firstName}} {{f.lastName}}</option>
-              {% endfor %}
-              </select>
+            <input class="form-control" type="text" id="hiddenFacilitatorArray" name="facilitators" value="" hidden>
+            <input
+              id="eventFacilitator"
+              class="form-control"
+              type="search"
+              placeholder="Search facilitator"
+              value="" />
+            <table class="table mb-3" id="facilitatorTable" name="facilitatorTable">
+              <thead>
+                <tr>
+                  <th scope="col">Facilitator</th>
+                  <th scope="col">Remove</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr hidden>
+                  <td>
+                    <p class="facilitatorName" required></p>
+                  </td>
+                  <td><button id="remove" type="button" class="btn btn-danger">Remove</button></td>
+
+                </tr>
+                {% for instructor in courseInstructor %}
```

Comment:
Facilitators aren't showing up on the create event page. I can see it was added by looking at View, but it's not there on Edit

Reply:
We made that a separate issue to keep this PR smaller. We'll get to that one once this one is merged.